### PR TITLE
add systemd consumer

### PR DESCRIPTION
**Pull request by relud**
Only builds with `-tags systemd`, and requires `cgo` and `systemd/sd-journal.h` to build successfully. Would've picked a pure-go reader for the systemd journal, but one doesn't seem to exist.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -35,6 +35,10 @@ aws:
 	@GOOS=linux GOARCH=amd64 $(BUILD_FLAGS) go build -o gollum
 	@zip -j dist/gollum_aws.zip gollum config/kinesis.conf dist/Procfile
 
+systemd:
```
Comment (arnecls) [inline makefile]: Please move linker / build options into the consumer file.
See https://github.com/trivago/gollum/blob/master/contrib/native/librdkafka/client.go#L17

**Comment by arnecls** [comment]
Plugins using native bindings should go to contrib/native.
Please move the consumer there.

**Comment by relud** [comment]
Will do

**Comment by relud** [comment]
moved to contrib/native

**Comment by arnecls** [comment]
Thanks a lot (as usual ^^).
:+1:

**Comment by andygrunwald** [comment]
We should mention the systemd consumer as well in the README

**Comment by arnecls** [comment]
done

**Pull request merged**